apps/src/templates/sectionAssessments/MultipleChoiceSurveyOverviewTable.jsx

```diff
@@ -241,22 +225,24 @@ class MultipleChoiceSurveyOverviewTable extends Component {
       sort: orderBy
     })(this.props.multipleChoiceSurveyData);
 
+    const questionIndex = this.state.selectedQuestionIndex;
```

Comment:
NIT: an (optional) shorthand for this is `const {selectedQuestionIndex} = this.state`

Reply:
lol. if i use that shorthand, the next line would wrap. so i wanted to give the value a shorter name while assigning. but if we like to be consistent about using that shorthand (which doesn't seem to be the case as far as i can tell) i'm happy to update this.